applications/serial_lte_modem/src/slm_util.c

```diff
@@ -93,32 +94,67 @@ int slm_util_at_scanf(const char *cmd, const char *fmt, ...)
 	return ret;
 }
 
-int slm_util_at_cmd_fwd_from_cb(char *buf, size_t len, char *at_cmd)
+static int terminate_at_response(char *buf, size_t len,
+				 const char *termination_str, int success_ret)
 {
-/* This gets called by interception callbacks invoked by nrf_modem_at_cmd().
+	len -= strlen(buf);
+	const int printf_ret = snprintf(buf + strlen(buf), len, "%s", termination_str);
+
+	if (printf_ret >= len) {
+		LOG_ERR("%u bytes of the AT response were truncated."
+			" The buffer needs to be made bigger.", printf_ret - len + 1);
+		return -NRF_E2BIG;
+	}
+	return success_ret;
+}
+
+int slm_util_at_cmd_no_intercept(char *buf, size_t len, const char *at_cmd)
+{
+	int ret;
+	char second_line[strlen("+CMx ERROR: xxxx")];
```

Comment:
When AT-command response returns some actual values, this expects that the following line (OK), fits in here. I take it that if there is multiple lines, then we just respond with ERROR, even if the AT-command itself was successful (as we did not receive OK)?

Reply:
Correct. If the response is longer than two lines, in which case the OK would not be in the first two lines.